resources/build/build-utils.sh

```diff
@@ -284,12 +284,19 @@ else
     COLOR_YELLOW=
     COLOR_RESET=
 fi
-###
+
+####################################################################################
+#
+# builder_ functions for standard build script parameter and process management
+#
+####################################################################################
 
 #
 # builder_ names are reserved.
 # _builder_ names are internal use and subject to change
 #
+_builder_default_action=build
+_builder_debug=true
```

Comment:
Given that the latter's a debug variable, something tells me it should be set to `false` when it comes time to merge.